crates/subspace-archiving/Cargo.toml

```diff
@@ -11,6 +11,7 @@ include = [
     "/Cargo.toml",
     "/README.md",
 ]
+readme = "README.md"
```

Comment:
I don't think this line is necessary, it is the default. I meant adding it to `include`, but looks like we already had it.